crates/biome_js_semantic/src/events.rs

```diff
@@ -96,7 +97,10 @@ pub enum SemanticEvent {
 
     /// Tracks where a symbol is exported.
     /// The range points to the binding that is being exported.
-    Exported { range: TextRange },
+    Exported {
+        range: TextRange,
+        declaration_start: TextSize,
+    },
```

Comment:
We should update the comment because it is outdated. The new field could be named `declared_at`